### PR TITLE
Separate code examples in object.proto docs

### PR DESCRIPTION
**Pull request by mrerberg**
### Description

Separate code examples in object.proto documentation for more readability. Also, refactored examples

### Motivation

Improves the readability of examples

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/object/proto/index.md [hunk 1]
```diff
@@ -87,26 +92,29 @@ const shapeC = new ShapeC();
...
-  console.log('myname');
+fn.prototype.myName = function () {
+  console.log('myName');
 };
 
 const obj = {};
-obj.__proto__ = fn.prototype;
-obj.myname(); // myname
+// obj.__proto__ = fn.prototype
+Object.setPrototypeOf(obj, fn.prototype);
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/object/proto/index.md:116]: This page is a demo for `__proto__`. We want to use `obj.__proto__ = fn.prototype`.

File: files/en-us/web/javascript/reference/global_objects/object/proto/index.md [hunk 2]
```diff
@@ -87,26 +92,29 @@ const shapeC = new ShapeC();
...
-Test.prototype.myname = function () {
-  console.log('myname');
+Test.prototype.myName = function () {
+  console.log('myName');
 };
 
-const a = new Test();
-console.log(a.__proto__ === Test.prototype); // true
-a.myname(); // myname
+const testChild = new Test();
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/object/proto/index.md:103]: It's not a `testChild`; it's a `testInstance`, or simply `test`.

File: files/en-us/web/javascript/reference/global_objects/object/proto/index.md (hunk 1 above)
Comment (mrerberg) [reply files/en-us/web/javascript/reference/global_objects/object/proto/index.md:116]: Done

File: files/en-us/web/javascript/reference/global_objects/object/proto/index.md (hunk 2 above)
Comment (mrerberg) [reply files/en-us/web/javascript/reference/global_objects/object/proto/index.md:103]: I would like to use `testInstance` to visually separate Test as constructor and test as instance

Comment (mrerberg) [reply files/en-us/web/javascript/reference/global_objects/object/proto/index.md:103]: Done

Comment (mrerberg) [reply files/en-us/web/javascript/reference/global_objects/object/proto/index.md]: Thx for comments!

**Review by Josh-Cena — APPROVED** [review]
This looks better. Thanks, and congratulations on your first contribution here—welcome aboard!

**Comment by mrerberg** [comment]
> This looks better. Thanks, and congratulations on your first contribution here—welcome aboard!

Thanks a lot for comments and congratulations!

**Pull request merged**